src/main/java/school/hei/patrimoine/visualisation/swing/ihm/google/GoogleSubmitScreen.java

```diff
@@ -13,27 +13,29 @@
 import lombok.extern.slf4j.Slf4j;
 import school.hei.patrimoine.google.GoogleApi;
 import school.hei.patrimoine.google.GoogleApi.GoogleAuthenticationDetails;
-import school.hei.patrimoine.visualisation.swing.ihm.google.modele.LinkedPatrimoine;
+import school.hei.patrimoine.visualisation.swing.ihm.google.modele.GoogleLinkList;
 import school.hei.patrimoine.visualisation.swing.ihm.google.modele.NamedString;
 
 @Slf4j
-public class GoogleDocsSubmitScreen {
+public class GoogleSubmitScreen {
   private final JFrame inputFrame;
   private final JPanel inputPanel;
-  private final JTextArea inputField;
+  private final JTextArea docsField;
+  private final JTextArea driveField;
   private final GoogleDocsLinkIdInputVerifier linkIdInputVerifier =
       new GoogleDocsLinkIdInputVerifier();
   private final GoogleApi googleApi;
   private final GoogleAuthenticationDetails authDetails;
 
-  public GoogleDocsSubmitScreen(GoogleApi googleApi, GoogleAuthenticationDetails authDetails) {
+  public GoogleSubmitScreen(GoogleApi googleApi, GoogleAuthenticationDetails authDetails) {
     this.googleApi = googleApi;
     this.authDetails = authDetails;
     inputFrame = newInputFrame();
     inputPanel = new JPanel();
     inputPanel.setLayout(new GridBagLayout());
 
-    inputField = new JTextArea(5, 70);
+    docsField = new JTextArea(5, 70);
```

Comment:
move docsField creation to a function because your constructor body is too heavy rn
same for driveField

Reply:
done